app/views/analyses/new/_phenotype_data_file_help.html.haml

```diff
@@ -0,0 +1,16 @@
+.alert.alert-info
```

Comment:
@Nuanda NOTE TO SELF: this is where I stopped reading the code. I'll start again from here.